README.md

```diff
@@ -7,4 +7,26 @@
 - stb_image
 - Imgui
 - tinyobjloader
-- spdlog
\ No newline at end of file
+- spdlog
+- googletest
+
+## Compile shaders
+[Install glslc](https://vulkan-tutorial.com/Development_environment#page_Shader-Compiler) on linux
+
+On linux
+```bash
+./compile_shaders.sh
+```
+
+## Build
+```bash
+mkdir cmake_build
+cmake -S . -B cmake_build
+cmake --build cmake_build
```

Comment:
In clion I can specify release or debug build in IDE, and cmake will generate two folders. But here you generate one build type, and without specs. I think we need add more configs to cmake then.

Reply:
https://stackoverflow.com/questions/27086145/what-is-the-default-build-configuration-of-cmake#:~:text=In%20this%20answer%2C%20it%20says,the%20default%20cmake%20build%20configuration.